src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  setTodo: (todo: Todo) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, setTodo }) => {
+  const [user, setUser] = useState({} as User);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    getUser(todo.userId).then(someUser => {
+      setUser(someUser);
+      setLoading(false);
```

Comment:
the same case
it's better to put setLoading to the finally block

Reply:
+